web/templates/recent_topic_row.hbs

```diff
@@ -57,10 +57,12 @@
                 </div>
                 <div class="recent_topic_actions">
                     <div class="recent_topics_focusable hidden-for-spectators">
-                        {{#if topic_muted}}
-                        <i class="zulip-icon zulip-icon-mute on_hover_topic_unmute recipient_bar_icon tippy-zulip-tooltip" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" data-tippy-content="{{t 'Unmute topic' }}" role="button" tabindex="0" aria-label="{{t 'Unmute topic' }}"></i>
+                        {{#if stream_muted}}
+                            <i class="zulip-icon zulip-icon-unmute {{#if topic_unmuted_in_muted_stream}}on_hover_topic_unmute {{else}}on_hover_topic_mute {{/if}} recipient_bar_icon tippy-zulip-tooltip" data-stream-id="{{stream_id}}" data-topic-name="{{topic}}" role="button" tabindex="0"
```

Comment:
`{{#if topic_unmuted_in_muted_stream}}on_hover_topic_unmute` doesn't make any sense -- the stream is already unmuted.